php-apache/usr/local/sbin/postfix.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+# call "postfix stop" when exiting
+trap "{ echo Stopping Postfix; /usr/sbin/postfix stop; exit 0; }" EXIT
+
+source /etc/sysconfig/network
+
+/usr/libexec/postfix/aliasesdb
+/usr/libexec/postfix/chroot-update
+
+if [ -n "${POSTFIX_RELAY_USER}" ]; then
```

Comment:
I vote for POSTFIX so as to match the service name, as SENDMAIL is incorrect term and serves only to confuse people further in regards to differences between sendmail software and sendmail cli interface